### PR TITLE
FEATURE: TNT-1405 SSF in MD

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Add selectionMode to MD types for Bear, Tiger and SPI

---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended check plugins`                                | Dashboard plugins tests                                    |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11880/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13904/ : SUCCESS in 16m 58s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11883/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13907/ : SUCCESS in 16m 55s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11884/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13908/ : SUCCESS in 17m 20s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11886/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13910/ : SUCCESS in 16m 33s

File: libs/sdk-model/src/dashboard/filterContext.ts [hunk 1]
```diff
@@ -41,6 +41,12 @@ export interface IDashboardAttributeFilterParent {
     };
 }
 
+/**
+ * Attribute filter selection mode value
+ * @alpha
```
Comment (ivanmjartan) [inline libs/sdk-model/src/dashboard/filterContext.ts:46]: define this tag as beta

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by ivanmjartan — APPROVED** [review]

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11892/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13916/ : FAILURE in 1m 49s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11893/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13917/ : SUCCESS in 16m 01s

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3539

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|6.1250s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|12.6200s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|5.9420s|
|✅|**dateFilter.spec.ts**|9|-|11.8630s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|15.8120s|
|✅|**dashboard.spec.ts**|9|-|62.7050s|
|✅|**attributeFilterButton.spec.ts**|13|-|47.7440s|
|✅|**dashboardDnD.spec.ts**|7|-|69.6480s|
|✅|**dashboardKPI.spec.ts**|8|-|188.2900s|
|✅|**drilling.spec.ts**|7(2 skipped)|-|58.8350s|
|✅|**headlineChartDrilling.spec.ts**|2|-|2.8950s|
|✅|**barChartDrilling.spec.ts**|4|-|5.5220s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|36.1310s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|17.7810s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|198.3950s|
|✅|**dashboardDateRelated.spec.ts**|8|-|67.1690s|
|✅|**dashboardDependentFilter.spec.ts**|8|-|119.7650s|
|✅|**attributeFilter.spec.ts**|1|-|6.0620s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|5.9130s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|5.9830s|
|✅|**pivotTableSizing.spec.ts**|1|-|3.8350s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul/424/).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul/50/ : FAILURE in 8m 15s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/392/ : FAILURE in 2m 11s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13920/ : SUCCESS in 23m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3665/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2508/ : SUCCESS in 11m 57s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul/424/ : SUCCESS in 29m 17s (non-voting)

**Pull request merged**